### PR TITLE
add QA test stories document

### PR DESCRIPTION
**Pull request by madvas**
### Summary

Adding document that should be followed by ops (and dev) guys when checking QA, before updates can be pushed into production.

### CODE REVIEW

File: qa-test-stories.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+- Challenge your meme.
+- Check if you've received email notification about your meme being challenged. 
+- Check if text of the email is correct.
+- After tx is confirmed, go to Vote page and check if your challenge is displayed there. 
+- Check challenge comment, challenger's address. 
+- Vote Stank for the meme with random amount. 
+- Switch to different address and vote Dank with larger amount than you voted Stank. Vote
+with random floating-point number (e.g 10.46).
+- Wait until reveal period starts and then reveal all your votes.
+- Wait until reveal period ends and swtich to Resolved Challenges tab.
```
Comment (fbielejec) [inline qa-test-stories.md]: switch

File: qa-test-stories.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+- Switch to address you voted Dank with. 
+- Check if all numbers related to voting are correct.
+- Claim your reward.
+- Check if your DANK balance was increased by the reward amount.  
+- Check if you've received email notification about reward amount. 
+- Check if text of the email is correct, including vote reward amount.
+- Switch to the address you voted Stank with.
+- Check if there's correctly no reward to claim.
+- Go to the detail page of the meme. 
+- Check if meme has correctly in-registry status.
```
Comment (fbielejec) [inline qa-test-stories.md]: correct

File: qa-test-stories.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- Wait until tx is confirmed.
+- Go to My Memefolio - Selling and check if your meme is there.
+- Go to Marketplace page. 
+- Switch sorting to Newest and check if your meme is on first position. 
+- Switch to "All Cards" grouping option and check if you see all cards you've offered.
+- Cancel offering of 1 card. 
+- After tx is confirmed, check if offering was correctly cancelled.
+- Switch to different address.
+- Go to My Memefolio - Collected and write down your rank and stats values for later comparison.
+- Buy on or more cards of your meme.
```
Comment (fbielejec) [inline qa-test-stories.md]: one or more

File: qa-test-stories.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+- Go to My Memefolio - Collected and check if previously written down values from My Memefolio - Collected were correctly updated.
+- Check if meme correctly appears in My Memefolio - Collected list. 
+- Go to Leaderboard - Collectors and check if rank and values match those from My Memefolio.
+- Check if you've received email notification about your meme being sold.
+- Check if text of the email is correct.
```
Comment (fbielejec) [inline qa-test-stories.md]: text + formatting?

**Comment by madvas** [comment]
> How do we verify all testing has been done before deploying? We could each time create an issue in kanban?

I guess we can tell ops guys to notify us on Slack when testing has been finished.

**Pull request merged**